projects/plugins/jetpack/modules/likes.php

```diff
@@ -24,12 +26,15 @@
 	'dns-prefetch'
 );
 
-include_once dirname( __FILE__ ) . '/likes/jetpack-likes-master-iframe.php';
-include_once dirname( __FILE__ ) . '/likes/jetpack-likes-settings.php';
+require_once dirname( __FILE__ ) . '/likes/jetpack-likes-master-iframe.php';
+require_once dirname( __FILE__ ) . '/likes/jetpack-likes-settings.php';
```

Comment:
```suggestion
require_once __DIR__ . '/likes/jetpack-likes-master-iframe.php';
require_once __DIR__ . '/likes/jetpack-likes-settings.php';
```
Supposed to be caught by a sniff, but upstream hasn't done a release yet to include https://phabricator.wikimedia.org/T273574.